readme.md

```diff
@@ -892,6 +892,7 @@
 - [HPC](https://github.com/dstdev/awesome-hpc#readme) - High Performance Computing.
 - [Geocaching](https://github.com/FoxFil/awesome-geocaching#readme) - Outdoor treasure-hunting activity that uses GPS-enabled devices.
 - [Regex](https://github.com/slevithan/awesome-regex#readme) - Specialized language for matching patterns in text.
+- [FOSS Alternatives](https://github.com/hardBSDk/awesome-alternatives#README) - A list of free and open-source alternatives for proprietary, freeware and shareware programs, games, web services and operating systems.
```

Comment:
I'd sauggest making the description more succint. Also, `#README` should be replaced with `#readme`, otherwise it doesn't really work.
```suggestion
- [FOSS Alternatives](https://github.com/hardBSDk/awesome-alternatives#readme) - Free and open-source alternatives for proprietary software.
```